scripts/pre-commit.mjs

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const chalk = require('chalk');
+import chalk from 'chalk';
```

Comment:
Update file to be like the following 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/scripts/pre-commit.mjs